pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -178,6 +178,21 @@ export default function(props: GraphDashboardProps) {
         />
       </Axis>
     </LineGraph>,
+
+    <LineGraph title="Snapshots Received" sources={storeSources}>
```

Comment:
nit: consider "Snapshot Data Received" or something, to better distinguish it from the snapshots graph above.

Reply:
done!